www/src/pages/components/alert.mdx

```diff
@@ -55,7 +55,7 @@ import SingleComponentStatus from '../../components/SingleComponentStatus';
 
 ##### Kitchen Sink
 
-```jsx live=true
+```jsx live
```

Comment:
`live=true` was being interpreted as a string, not a boolean. Fixing this everywhere.